en/integrations/event-management/sc-bsm.md

```diff
@@ -3,124 +3,255 @@ id: sc-hp-bsm
+ "</event_data>"
+}
 ```
 
-The Stream Connector is not loaded anymore!
+### Custom event format
+
+This stream connector allows you to change the format of the event to suit your needs. Only the **event** part of the json is customisable. It also allows you to handle events type that are not handled by default such as **ba_status events**.
+
+In order to use this feature you need to configure a json event format file and add a new stream connector parameter.
+
+| Type   | Name        | Value                                                   |
+| ------ | ----------- | ------------------------------------------------------- |
+| string | format_file | /etc/centreon-broker/lua-conf/bsm-events-format.json    |
 
-9. Optionally, you can even delete the script file:
+> The event format configuration file must be readable by the centreon-broker user
 
-```bash
-rm -f /usr/share/centreon-broker/lua/bsm_connector.lua
+To learn more about custom event format and templating file, head over the following **[documentation](https://github.com/centreon/centreon-stream-connector-scripts/blob/master/modules/docs/templating.md#templating-documentation)**.
+
+## Curl commands
+
+Here is the list of all the curl commands that are used by the stream connector.
+
+### Send events
+
+You can trigger a signal with the following command:
+
+```shell
+curl -X POST -H 'content-type: application/xml' 'https://centreon.bsm.server:30005/bsmc/rest/events/myCentreon/' -d '{"<event_data>" "<hostname>"  'srv-vp-central01'  "</hostname>" \
+        "<svc_desc>"  'Swap'  "</svc_desc>" \ 
+        "<state>" 'Critical'  "</state>" \
+        "<last_update>" '12/16/2021 1:14 PM'  "</last_update>" \ 
+        "<output>"  'Critical: Swap Total: 1.60 GB Used: 1.51 GB (94.44%) Free: 1.25 MB (5.56%)\n'  "</output>" \
+        '0' \
+        "<url>"  'no action url for this host'  "</url>" \
+        "<source_host_id>" '3450'  "</source_host_id>" \
+        "<source_svc_id>"  '1245'  "</source_svc_id>" \
+        "<scheduled_downtime_depth>"  '0'  "</scheduled_downtime_depth>" \
+ "</event_data>"}'
 ```
 
+You can then close this signal with the following command:
+
+```shell
+curl -X POST -H 'content-type: application/xml' 'https://centreon.bsm.server:30005/bsmc/rest/events/myCentreon/' -d '{"<event_data>" "<hostname>"  'srv-vp-central01'  "</hostname>" \
+        "<svc_desc>"  'Swap'  "</svc_desc>" \
+        "<state>" 'OK'  "</state>" \ 
+        "<last_update>" '12/16/2021 1:45 PM'  "</last_update>" \
+        "<output>"  'OK: Swap Total: 1.60 GB Used: 91.25 MB (5.56%) Free: 1.51 GB (94.44%)\n'  "</output>" \
+        '0' \
+        "<url>"  'no action url for this host'  "</url>" \
+        "<source_host_id>" '3450'  "</source_host_id>" \
+        "<source_svc_id>"  '1245'  "</source_svc_id>" \
+        "<scheduled_downtime_depth>"  '0'  "</scheduled_downtime_depth>" \
+ "</event_data>"}'
+```
```

Comment:
there is too much things from another documentation to feel like it is the curl that is send. This is the hardest part when refactoring a stream connector. You have to guess the curl command from the code.